### ISSUE TITLE
the tactics of territorial aggression

### ISSUE THREAD

**Issue by one-sh** [WarEmu/WarBugs#14758]
it seems that the squig herders has some problems with the tactics of territorial aggression. For the first time, when you insert it into the tactical slot, it works correctly, but when you die, after that, sometimes you can summon a pet. after that, you will have all the advantages of this tactic (+ 25% increased DPS, squig leap without delay) and other skills related to the pet (disarming, knocking down, git em ).

**Comment by sioding** [comment]
Duplicate of #13423

**Comment by one-sh** [comment]
I meant that when you have a tactic established, you can summon a battle squig after u died first time, while a tactic prohibits it. and you also can use the battle squig related skills although the tactic prohibits it, and to have all advantages related this tactic.

**Comment by grunbag** [comment]
When does it happen ?

**Comment by one-sh** [comment]
It happened yesterday when I was doing a PQ. after when i was died and res, I clicked on the ability to summon a pet and it appeared, although I had this tactic installed. then I realized that the ability to summon a pet with established TA tactic appears when you die first time. if u change tactics it will be work as intended again. And it happened repeatedly last evening. I also tried to act today, but today for some reason it didn’t work out anymore. but some differences today i was trying it in rvr zone, yesterday it happened in pve.
